xgi/classes/reportviews.py

```diff
@@ -171,104 +148,38 @@ def __getitem__(self, n):
             the node does not exist in the hypergraph.
 
         """
-        if isinstance(n, slice):
-            raise XGIError(
-                f"{type(self).__name__} does not support slicing, "
-                f"try list(H.nodes)[{n.start}:{n.stop}:{n.step}]"
-            )
-        elif n not in self._nodes:
-            raise XGIError(f"The node {n} is not in the hypergraph")
-
-        return self._node_attr[n]
+        try:
+            return self._id_attrs[id]
+        except:
```

Comment:
Comment: it's generally a bad idea to catch every single Exception using a naked `except`. It's always recommended to specify the exception type. In this case it'd be `except KeyError` (I believe).